fdbclient/NativeAPI.actor.cpp

```diff
@@ -1577,7 +1581,8 @@ DatabaseContext::DatabaseContext(Reference<AsyncVar<Reference<IClusterConnection
 		    SpecialKeySpace::IMPLTYPE::READWRITE,
 		    std::make_unique<ClientProfilingImpl>(
 		        KeyRangeRef(LiteralStringRef("profiling/"), LiteralStringRef("profiling0"))
-		            .withPrefix(SpecialKeySpace::getModuleRange(SpecialKeySpace::MODULE::MANAGEMENT).begin)));
+		            .withPrefix(SpecialKeySpace::getModuleRange(SpecialKeySpace::MODULE::MANAGEMENT).begin)),
+		    /* depracated */ 720);
```

Comment:
```suggestion
		    /* deprecated */ 720);
```

Reply:
Cool, I will fix this.
yeah, I think we can close it once this is merged.
Thanks.

Reply:
Done.
Just found this cool feature `devDocPreview` you added, so nice!